kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -233,6 +223,28 @@
   private static final String TABLE_CREATE_DOC =
           "Automatically create BigQuery tables if they don't already exist";
 
+  public static final String AUTO_CREATE_BUCKET_CONFIG =                "autoCreateBucket";
+  private static final ConfigDef.Type AUTO_CREATE_BUCKET_TYPE =          ConfigDef.Type.BOOLEAN;
+  public static final Boolean AUTO_CREATE_BUCKET_DEFAULT =                true;
+  private static final ConfigDef.Importance AUTO_CREATE_BUCKET_IMPORTANCE = ConfigDef.Importance.MEDIUM;
+  private static final String AUTO_CREATE_BUCKET_DOC =
+          "Whether to automatically create the given bucket, if it does not exist";
+
+  public static final String ADD_NEW_BQ_FIELDS_CONFIG =                 "addNewBigQueryFields";
+  private static final ConfigDef.Type ADD_NEW_BQ_FIELDS_TYPE =          ConfigDef.Type.BOOLEAN;
+  public static final Boolean ADD_NEW_BQ_FIELDS_DEFAULT =               false;
+  private static final ConfigDef.Importance ADD_NEW_BQ_FIELDS_IMPORTANCE = ConfigDef.Importance.MEDIUM;
+  private static final String ADD_NEW_BQ_FIELDS_DOC =
+          "If true, new fields can be added to BigQuery tables during subsequent schema updates"
+          + "and would be NULLABLE";
+
+  public static final String CHANGE_REQ_BQ_FIELDS_TO_NULLABLE_CONFIG =   "changeRequiredFieldsToNullable";
```

Comment:
This is a confusing name. We have a separate config (`allBQFieldsNullable`) that forces all required fields to nullable. Something like `allowRequiredFieldRelaxation` or something.

Reply:
Related: What if `allBQFieldsNullable=true`, and `allowRequiredFieldRelaxation=false`? This seems like a irresolvable conflict--the connector would be configured to set all fields to nullable, but also is configured not to allow that.

Reply:
We could validate this in the `validate` method probably